confluent_kafka/serializing_producer.py

```diff
@@ -152,16 +153,23 @@ def produce(self, topic, key=None, value=None, partition=-1,
                 the application should call :py:func:`SerializingProducer.Poll`
                 and try again.
 
-             KafkaException: for other errors, see exception code
+             KafkaError: For all other errors
```

Comment:
Add explicit mentions of Key|ValueSerializerError here, finish off with KafkaError for everything else.